### PR TITLE
Update Minio chart to latest release RELEASE.2018-02-09T22-40-05Z

### PR DESCRIPTION
**Pull request by nitisht**
Update Minio Chart to use latest Minio release `RELEASE.2018-02-09T22-40-05Z`

### CODE REVIEW

**Review by paulczar — APPROVED** [review]
LGTM

File: stable/minio/values.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Distributed Minio ref: https://docs.minio.io/docs/distributed-minio-quickstart-guide
 ##
 image: "minio/minio"
-imageTag: "RELEASE.2018-01-18T20-33-21Z"
+imageTag: "RELEASE.2018-02-09T22-40-05Z"
 imagePullPolicy: "Always"
```
Comment (unguiculus) [inline stable/minio/values.yaml]: Could you change `imagePullPolicy` to `IfNotPresent`?

Comment (nitisht) [reply stable/minio/values.yaml]: good point @unguiculus but I wanted to keep this as `Always` just to make sure all the deployments run the same and latest Minio release. 

Given the Minio image is just 11MB, IMHO this is not a very large load on users

Comment (unguiculus) [reply stable/minio/values.yaml]: Always only makes sense for `latest` unless you update an existing tag.

Comment (nitisht) [reply stable/minio/values.yaml]: > Always only makes sense for latest 

I did not get @unguiculus could you please elaborate..

Comment (unguiculus) [reply stable/minio/values.yaml]: Always doesn't make sense for stable image that don't change. There's no point in always trying to pull the image again. Please change it to `IfNotPresent`.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by nitisht** [comment]
@unguiculus done. PTAL

File: stable/minio/README.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ The following tables lists the configurable parameters of the Minio chart and th
 | Parameter                  | Description                         | Default                                                 |
 |----------------------------|-------------------------------------|---------------------------------------------------------|
 | `image`                    | Minio image name                    | `minio/minio`                                           |
-| `imageTag`                 | Minio image tag. Possible values listed [here](https://hub.docker.com/r/minio/minio/tags/).| `RELEASE.2018-01-18T20-33-21Z`|
+| `imageTag`                 | Minio image tag. Possible values listed [here](https://hub.docker.com/r/minio/minio/tags/).| `RELEASE.2018-02-09T22-40-05Z`|
 | `imagePullPolicy`          | Image pull policy                   | `Always`                                                |
```
Comment (unguiculus) [inline stable/minio/README.md]: Please update.

Comment (nitisht) [reply stable/minio/README.md]: Good catch, updated

Comment (unguiculus) [reply stable/minio/README.md]: Not quite done yet.

File: stable/minio/README.md [hunk 3]
```diff
@@ -75,8 +75,8 @@ The following tables lists the configurable parameters of the Minio chart and th
 | Parameter                  | Description                         | Default                                                 |
 |----------------------------|-------------------------------------|---------------------------------------------------------|
 | `image`                    | Minio image name                    | `minio/minio`                                           |
-| `imageTag`                 | Minio image tag. Possible values listed [here](https://hub.docker.com/r/minio/minio/tags/).| `RELEASE.2018-01-18T20-33-21Z`|
-| `imagePullPolicy`          | Image pull policy                   | `Always`                                                |
+| `imageTag`                 | Minio image tag. Possible values listed [here](https://hub.docker.com/r/minio/minio/tags/).| `RELEASE.2018-02-09T22-40-05Z`|
+| `imagePullPolicy`          | Image pull policy                   | `IfNotPresent`                                          |
```
Comment (nitisht) [inline stable/minio/README.md:79]: It is done here @unguiculus

**Review by unguiculus — APPROVED** [review]

**Pull request merged**